src/main/java/frc/robot/subsystems/Ramp.java

```diff
@@ -38,35 +38,23 @@ public void periodic() {
             SmartShuffleboard.put("Ramp", "Desired pos", rampPos);
             SmartShuffleboard.put("Ramp", "Reverse Switch Tripped", getReversedSwitchState());
             SmartShuffleboard.put("Ramp", "Forward Switch Tripped", getForwardSwitchState());
-//            double pidP = SmartShuffleboard.getDouble("Ramp", "PID P", neoPidMotor.getPidController().getP());
```

Comment:
The code above (setting thePID_FF) can probably be be part of a command rather then run every time in periodic

Reply:
Since the ramp is a PID and the MoveRamp command finishes instantly, what do you recommend? Reading the document it does say to use the method "Infrequently" because it takes time. I could store the last PID Value and compare against it before setting it?

Reply:
Is there a functional reason that the command be done immediately? I would think that it would make sense for a command to terminate once the ramp it at the position.
So you can have a sequential group of two commands: One to get the ramp "close enough" with FF and another to get the ramp closer (without the FF), and the second one to end once the ramp error is within range.

Reply:
I checked and I don't _think_ it will break anything if I make the ramp move not end immediately.  I added the logic to the ramp commands